pkg/app/AppService.go

```diff
@@ -773,9 +776,9 @@ func (impl *AppServiceImpl) UpdatePipelineStatusTimelineForApplicationChanges(ap
 	return isTimelineUpdated, isTimelineTimedOut, kubectlApplySyncedTimeline, nil
 }
 
-func (impl *AppServiceImpl) WriteCDSuccessEvent(appId int, envId int, override *chartConfig.PipelineOverride) {
-	event, _ := impl.eventFactory.Build(eventUtil.Success, &override.PipelineId, appId, &envId, eventUtil.CD)
-	impl.logger.Debugw("event WriteCDSuccessEvent", "event", event, "override", override)
+func (impl *AppServiceImpl) WriteCDNotificationEvent(appId int, envId int, override *chartConfig.PipelineOverride, eventType eventUtil.EventType) {
```

Comment:
would prefer this in deploymentEventHandler separately as appService is polluted with too many methods